examples/http-compression/http_compression.client.out

```diff
@@ -0,0 +1,61 @@
+# To invoke the autoCompress Service, use following curl command.
+$ curl -v --output - http://localhost:9090/autoCompress -H "Accept-Encoding:deflate"
+> GET /autoCompress HTTP/1.1
+> Host: localhost:9090
+> User-Agent: curl/7.58.0
+> Accept: */*
+> Accept-Encoding:deflate
+>
+< HTTP/1.1 200 OK
+< content-type: application/json
+< content-encoding: deflate
+< content-length: 41
+<
+x��V
+* Connection #0 to host localhost left intact
+�,HU�Rr,-�WH��-(J-.���S�����	�
+
+
+# To invoke the autoCompress Service, use following curl command.
```

Comment:
Shall we mention the difference between this and the above curl in this comment?